Spigot-Server-Patches/0690-Implement-Keyed-on-World.patch

```diff
@@ -64,19 +64,19 @@ index 760579921927b4c8b0f20b2611b95fd626e4b27f..3075700dfa992da81b10246fcf7c7ad1
          return this.c;
      }
 diff --git a/src/main/java/org/bukkit/craftbukkit/CraftServer.java b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
-index cbac3c96c5d3c1551912f5769bfc50d690519495..03b8d67a5f0088c0254b2099f27e8dcae32a6221 100644
+index e8305df0ce11bf7c297bf5f0acc95f07324e4143..51d69406e22df2af777bed6982b3f9c960458e91 100644
 --- a/src/main/java/org/bukkit/craftbukkit/CraftServer.java
 +++ b/src/main/java/org/bukkit/craftbukkit/CraftServer.java
-@@ -1154,7 +1154,7 @@ public final class CraftServer implements Server {
-             chunkgenerator = worlddimension.c();
+@@ -1161,7 +1161,7 @@ public final class CraftServer implements Server {
+         } else if (name.equals(levelName + "_the_end")) {
+             worldKey = net.minecraft.world.level.World.THE_END;
+         } else {
+-            worldKey = ResourceKey.a(IRegistry.L, new MinecraftKey(name.toLowerCase(java.util.Locale.ENGLISH)));
++            worldKey = ResourceKey.newResourceKey(IRegistry.getWorldRegistry(), new net.minecraft.resources.MinecraftKey(creator.key().getNamespace().toLowerCase(java.util.Locale.ROOT), creator.key().getKey().toLowerCase(java.util.Locale.ROOT))); // Paper
```

Comment:
English locale

Reply:
oh yeah, I always forget to change it from what intellij autocompletes to.

Reply:
Done